tests/js/ESLintBearTest.py

```diff
@@ -0,0 +1,51 @@
+
+from bears.js.ESLintBear import ESLintBear
+from tests.LocalBearTestHelper import verify_local_bear
+from coalib.misc.ContextManagers import prepare_file
+
+
+test_good = """function addOne(i) {
+    if (!isNaN(i)) {
+        return i+1;
+    }
+    return i;
+}
+
+addOne(3);
+
+""".splitlines(True)
+
+test_bad = """function addOne(i) {
+    if (i != NaN) {
+        return i ++
+    }
+    else {
+        return
+    }
+};
+""".splitlines(True)
+
+config = """{
+    "rules": {
+        "indent": [1, 4]
+    }
+}
+""".splitlines(True)
+
+eslintconfig = """{
+    "extends": "eslint:recommended",
+    "rules": {
+        "consistent-return": 2,
+        "indent"           : [1, 4],
+        "no-else-return"   : 1,
+        "semi"             : [1, "always"],
+        "space-unary-ops"  : 2
+    }
+}
+""".splitlines(True)
+
+ESLintBearGoodTest = verify_local_bear(ESLintBear,
+                                       valid_files=(test_good,),
+                                       invalid_files=(test_bad,),
+                                       settings={"eslint_config":
```

Comment:
Im still wondering about the config ... Are you sure it makes sense ???
Youre giving a list as a cli arg. I have no clue how it works ...